README.md

```diff
@@ -1,126 +1,86 @@
-    1. Fly the pipeline.
-    1. Verify the pipeline runs successfully.
-
-## Teardown procedure
-1. Delete all deployments managed by bosh: (development or staging and production, tooling, master)
-    1. Run `bosh -d ${deployment_name} delete-deployment` for all deployments in `bosh deployments`
-1. Bring up a bootstrap concourse instance as described above:
-    1. Run `01-bootstrap-terraform.sh`, `02-bootstrap-concourse.sh`, `03-main-terraform.sh`
-    1. If your bootstrap concourse already has peering to tooling VPC, run `01-bootstrap-terraform.sh` again to disable.
-1. From the web UI in your bootstrap concourse, run `terraform-provision/destroy-*` for development, or staging and production.
-1. Run `destroy-tooling` in the `terraform-provision` pipeline.
-1. Teardown bootstrap concourse and terraform stack: `./scripts/bootstrap/teardown.sh`
-1. Delete the `${TF_STATE_BUCKET}`, `${VARZ_BUCKET}`, `${SEMVER_BUCKET}`, and `${BOSH_RELEASES_BLOBSTORE_BUCKET}` through the AWS console.
-
-## Validating `terraform plan` locally (beta)
-
-* Locally clone git@github.com:18F/cg-pipeline-tasks.git; you'll need `cg-pipeline-tasks/terraform-apply.sh`
-* This terraform setup won't use your AWS profile, so export your AWS secret
-  key and access key (e.g. w/
-  https://github.com/pburkholder/Dotfiles/blob/master/bin/aws_emit)
-* Set up a temp directory and get the env vars from, say, the tooling pipeline:
-```
-export TF_STACK=$(pwd)
-mkdir tmp/
-cd tmp/
-
-# set env vars from pipeline as `export key=value` for plan parameters
-eval "$(fly -t fr get-pipeline -p terraform-provision -j | jq -r '.| .jobs[] | select(.name=="plan-bootstrap-tooling") | .plan[1].params | to_entries[] | "export " + .key + "=" + .value')"
-
-# export your AWS vars, then run:
-/your/path/to/cg-pipeline-tasks/terraform-apply.sh
-
-# clean up:
-cd .. & rm -rf tmp/
-```
+# Cloud.gov Provisioning System
+
+This repository holds the terraform configuration (and BOSH vars and ops-files)
+to bootstrap our infrastructure.
+
+## Layout
+
+### Terraform
+
+This system is using Terraform 0.11.  Future versions of the main providers
+(AWS in particular) [will not work with this
+version](https://www.hashicorp.com/blog/deprecating-terraform-0-11-support-in-terraform-providers/).
+At some point, we'll need to upgrade to the (much better) 0.12 syntax.
+
+The main terraform directories are:
+
+* `modules`: where we decompose our configuration into [Terraform
+  modules](https://www.terraform.io/docs/configuration-0-11/modules.html)
+* `modules/stack/base` & `modules/stack/spoke`: the main modules that define
+  the bulk of each environment
+* `stacks`: the various "environments"
+
+#### Environments
+
+The `main` stack is a template that is used to provision the production,
+staging, and development "environments."
+
+> ##### Aside: Multiple environments in one account
+>
+> Why do I keep putting "environments" in quotes?  One confusing and suboptimal
```

Comment:
I think this can live somewhere else, we don't want this information being broadcast while we're in this state.

Reply:
I see your point, but I'm not sure obscuring this fact will outweigh the benefit of making the configuration clear to the engineer tasked with working in this repository.  How strongly do you feel about this?

Reply:
Very strongly. This type of commentary belongs in internal documentation where we can debate it freely, not public readmes.

Reply:
Done